pkgs/applications/networking/cluster/k3s/default.nix

```diff
@@ -84,11 +95,11 @@ let
     sha256 = k3sRootSha256;
     stripRoot = false;
   };
-  k3sPlugins = buildGoPackage rec {
-    name = "k3s-cni-plugins";
+  k3sCNIPlugins = buildGoModule rec {
```

Comment:
upstream uses "GO111MODULE=off" for this ([see here](https://github.com/k3s-io/k3s/blob/5fb370e53e0014dc96183b8ecb2c25a61e891e76/scripts/build#L100)), but I think building it with go modules shouldn't make a difference. :man_shrugging: